receiver/zookeeperreceiver/utils.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zookeeperreceiver
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+)
+
+func sendCmd(conn net.Conn, cmd string) *bufio.Scanner {
+	fmt.Fprintf(conn, "%s\n", cmd)
```

Comment:
Nit: Goland will flag this as a warning: 'unhandled error'. Instead `_, _ = fmt.Fprintf(conn, "%s\n", cmd)`

Reply:
Will update to handle the error.